benchmark/transaction/seqscan_benchmark.cpp

```diff
@@ -0,0 +1,152 @@
+#include <common/scoped_timer.h>
```

Comment:
Should probably make a new folder called `execution` rather than have it in `transaction`.